.circleci/config.yml

```diff
@@ -1231,10 +1231,10 @@ workflows:
                 at: /
           workflow-name: "GCP-Daily-Services-Recovery-4N-1C"
 
-  AWS-Daily-Services-Comp-Basic-Performance-4N-4C:
+  AWS-Daily-Services-Comp-Restart-Performance-Random-4N-4C:
```

Comment:
Just a note, PR containing these changes are yet to be merged in platform-regression by @ljianghedera . So these tests will not run until it is merged.